spec/helpers/organisation_helper_spec.rb

```diff
@@ -1,15 +1,15 @@
-require "test_helper"
+require "rails_helper"
 
 describe OrganisationHelper do
   include OrganisationHelpers
```

Comment:
I had to manually copy both of these files from `test/support`. That seems reasonable, since we'll need to migrate all the helpers over anyway.

Reply:
Ah yep sorry. The OrganisationHelpers module is used in specs that aren't yet converted, ie integration specs. Rather than migrate them over, I'll add `Dir[Rails.root.join("test/support/**/*.rb")].sort.each { |f| require f }` in addition to Dir[Rails.root.join("spec/support/**/*.rb")].sort.each { |f| require f }` for the time being, so we don't end up with two copies of the same file that might then diverge.

Reply:
Sounds good - I couldn't decide whether it would be better to copy or reference. Time will tell 🤞.